state/metrics.go

```diff
@@ -34,15 +34,16 @@ type MetricBatch struct {
 }
 
 type metricBatchDoc struct {
-	UUID        string    `bson:"_id"`
-	ModelUUID   string    `bson:"model-uuid"`
-	Unit        string    `bson:"unit"`
-	CharmURL    string    `bson:"charmurl"`
-	Sent        bool      `bson:"sent"`
-	DeleteTime  time.Time `bson:"delete-time"`
-	Created     time.Time `bson:"created"`
-	Metrics     []Metric  `bson:"metrics"`
-	Credentials []byte    `bson:"credentials"`
+	UUID           string    `bson:"_id"`
+	ModelUUID      string    `bson:"model-uuid"`
+	Unit           string    `bson:"unit"`
+	CharmURL       string    `bson:"charmurl"`
+	Sent           bool      `bson:"sent"`
+	DeleteTime     time.Time `bson:"delete-time"`
+	Created        time.Time `bson:"created"`
+	Metrics        []Metric  `bson:"metrics"`
+	Credentials    []byte    `bson:"credentials"`
+	SLACredentials []byte    `bson:"sla-Credentials"`
```

Comment:
`bson:"sla-credentials,omitempty"`     if not, then at least lowercase the Credentials in the bson tag